main.js

```diff
@@ -2,3 +2,67 @@
 // Go back to the lesson if you feel lost
 // Read the docs well, they usually have pretty well documented examples
 // Consider making your fetch work with a button
+
+const img =document.createElement('img');
+const button = document.querySelector('button');
+const apiUrl = `https://rickandmortyapi.com/api/character/[1,2,3,4,5]`;
+// const numItemsToGenerate = 1;
+
+const getRickAndMorty =() =>{
+    axios({
+        url:apiUrl ,
+        method:'get',
+    })
+        .then( res=>{
+            console.log(res);
+            res.data.forEach(element => {
+                const imgUrl=element.image;
```

Comment:
```suggestion
               const img =document.createElement('img');
                 const imgUrl=element.image;
```
You can create the image element in the loop, So with each loop, you create a new Image tag and set the SRC for the image.